rs/messaging/src/state_machine.rs

```diff
@@ -109,12 +109,12 @@ impl StateMachine for StateMachineImpl {
                 .observe_no_canister_allocation_range(&self.log, message);
         }
 
-        // Time out requests.
-        let timed_out_requests = state.time_out_requests();
+        // Time out expired messages.
+        let timed_out_messages = state.time_out_requests();
```

Comment:
We will be renaming it in #879. For now, it's less confusing for the method to still be named `time_out_requests()` (which is more accurate) while the metric can already use "messages" instead of "requests". It's still going to be "requests" even after we merge the new `CanisterQueues` until we actually enable best-effort messages on mainnet.

Reply:
That's fine; just for clarity, do you plan to later on initiate the timing out of (best-effort response) messages from this method, or from somewhere else?